node/src/accountant/mod.rs

```diff
@@ -1917,7 +1403,7 @@ mod tests {
             hash
         ));
         log_handler.exists_log_containing(
-            r#"WARN: Accountant: Failed transaction with a hash '0x0000000000000000000000000000000000000000000000000000000000003039' but without the record - thrown out"#,
+            r#"WARN: Accountant: Failed transaction with a hash '0x0000…3039' but without the record - thrown out"#,
```

Comment:
We first worried a bit about the security of revealing full tx hashes. Later we just stopped talking about it. And even more later, we said that we will prefer to print the hashes in the full length so please change it accordingly.